core/dbt/cli/params.py

```diff
@@ -90,6 +90,12 @@
     is_flag=True,
 )
 
+empty = click.option(
+    "--empty/--no-empty",
```

Comment:
what does `--no-empty` mean?

Reply:
Ah, unfortunately it's necessary because of the way that `dbt retry` reconstructs the previous command's args. It sees an `empty:False` and tries to map that to a non-existent `--no-empty` flag explicitly, causing failures.